src/main/cms/cms_menu_imu.c

```diff
@@ -886,15 +886,31 @@ static CMS_Menu cmsx_menuFilterPerProfile = {
 static uint8_t cmsx_dstPidProfile;
 static uint8_t cmsx_dstControlRateProfile;
 
-static const char * const cmsx_ProfileNames[] = {
+#if PID_PROFILE_COUNT >= CONTROL_RATE_PROFILE_COUNT
+#define PROFILE_COUNT PID_PROFILE_COUNT
+#else
+#define PROFILE_COUNT CONTROL_RATE_PROFILE_COUNT
+#endif
+
+static const char * const cmsx_ProfileNames[PROFILE_COUNT+1] = {
     "-",
     "1",
     "2",
     "3"
+#if PROFILE_COUNT >= 4
+    ,"4"
```

Comment:
I suggest using "...", (comma in the end for all entries)
as it is more readable. C allows the comma after the last entry, so that's not a problem.

Reply:
No need to figure out the max number of profiles... you could simply define the names for 6, as thats the maximum